src/simsopt/objectives/least_squares.py

```diff
@@ -162,7 +162,7 @@ def unweighted_residuals(self, x=None, *args, **kwargs):
                     break
 
                 output = np.array([out]) if not np.ndim(out) else np.asarray(out)
-                output -= self.goals[i]
+                output = output - self.goals[i]
                 if self.first_eval:
```

Comment:
Why this change?

Reply:
I've had this break on a few occasions with more general optimization metrics. I can't reproduce it with the tests but the error is usually printed as
`gcc12 error: no match for 'operator*' overload operator not working`